app-dicts/myspell-en/myspell-en-20181101.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+MY_PV="${PV:0:4}-${PV:4:2}.${PV:6:2}"
+
+inherit myspell-r2
+
+DESCRIPTION="English dictionaries for myspell/hunspell"
+HOMEPAGE="https://extensions.libreoffice.org/extensions/english-dictionaries"
+SRC_URI="https://extensions.libreoffice.org/extensions/english-dictionaries/${MY_PV}/@@download/file/dict-en-${PV}.oxt"
+
+LICENSE="BSD MIT LGPL-3"
```

Comment:
Still incorrect.

Reply:
are you being vague on purpose?

Reply:
Maybe I'm just tired of retyping the same thing over and over again. It is really discouraging to see that most of the contributors can't distinguish 'LGPL-3' from 'LGPL-3+', or just never bother looking what other options we have.